ixmp/core.py

```diff
@@ -1102,14 +1102,18 @@ def clone(self, model=None, scenario=None, annotation=None,
                 ' release, please use `scenario`')
             scenario = kwargs.pop('scen')
 
+        if first_model_year is not None:
```

Comment:
Per discussion with @danielhuppmann, this is not actually MESSAGE-specific; and in any case MESSAGE should not be referenced in ixmp. Remove these lines.

Reply:
@khaeru it somehow contradicts with another comment about removing `clone` method from `massage_ix`, doesn't it?

imo we should:
- remove code duplication from `message_ix` `clone` method (adding call of `ixmp.Scenario.clone` there)
- move this timeseries copying stuff to `message_ix.Scenario.clone` method

Can you confirm?

Reply:
[Discussed on Slack]

The conclusion was:
- there is nothing at all for `message_ix.Scenario.clone()` to do, at the moment. So I will (in a separate PR) remove it altogether, which means this parent class method, `ixmp.Scenario.clone()`, will be called.
- this parameter is not only for MESSAGE scenarios; it's for any `ixmp.Scenario` with time series data. So the docstring doesn't need to refer to MESSAGE.

Reply:
Warning removed